dotnet/src/SemanticKernel.Core/Data/VolatileVectorStoreRecordCollection.cs

```diff
@@ -227,7 +227,7 @@ public async IAsyncEnumerable<VectorSearchResult<TRecord>> VectorizedSearchAsync
 
         if (vector is not ReadOnlyMemory<float> floatVector)
         {
-            throw new NotSupportedException($"The provided vector type {vector.GetType().FullName} is not supported by the Qdrant connector.");
+            throw new NotSupportedException($"The provided vector type {vector.GetType().FullName} is not supported by the Volatile Vector Store.");
```

Comment:
Would `nameof` be better so we don't have to make updates when we change the class name?

Reply:
All the connectors use the more friendly description rather than nameof at the moment.  I can do a separate pass through all if we really want to change this, but I'd say it's more valuable if we want to make changes to this to do it then.